uvloop/sslproto.pyx

```diff
@@ -699,34 +834,18 @@ class SSLProtocol(object):
                 'protocol': self,
             })
 
-    def _finalize(self):
-        self._sslpipe = None
-
-        if self._transport is not None:
-            self._transport.close()
-
-    def _abort(self):
-        try:
-            if self._transport is not None:
-                self._transport.abort()
-        finally:
-            self._finalize()
 
+cdef _add_water_defaults(high, low, kb):
```

Comment:
Can you put this function elsewhere and call it from `UVBaseTransport._set_write_buffer_limits()`?

Reply:
Sure thing!